ext/mixed/css/display.css

```diff
@@ -88,6 +88,19 @@ ol, ul {
     user-select: none;
 }
 
+#query-parser {
+    margin-top: 10px;
+    font-size: 24px;
+}
+
+.query-parser-term {
+    margin-right: 5px;
+}
+
+html:root[data-yomichan-page=search] body {
+    min-height: 101vh; /* always show scroll bar to avoid scanning problems */
```

Comment:
I think ```overflow-y: scroll;``` accomplishes what you want.

Reply:
Yes, it seems to do what I was trying to do here. However, in a situation where you have already scrolled a bit down on a long result and scan text that returns a short result, only `padding-bottom: 100vh;` or similar helps.